src/json.cpp

```diff
@@ -0,0 +1,3 @@
+#ifdef BOOST_ALL_NO_LIB
+#    include <boost/json/src.hpp>
+#endif
```

Comment:
The include should be able to be outside of the `ifdef` - with this ifdef, boost/json is only header only on Windows systems.